### PR TITLE
chore: add dockerfile

### PR DESCRIPTION
**Pull request by tokers**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM golang:1.14.10 AS build-env
+
+RUN rm -rf /etc/localtime \
+    && ln -s /usr/share/zoneinfo/Hongkong /etc/localtime \
```
Comment (nic-chen) [inline Dockerfile:4]: why we use hongkong timezone here..

**Review by membphis — COMMENTED** [review]
bad pr title style

File: Dockerfile (hunk 1 above)
Comment (tokers) [reply Dockerfile:4]: All dockerfiles in our products use this time zone, we can also set it to Asia/Shanghai.

**Review by membphis — APPROVED** [review]
LGTM

**Review by nic-chen — APPROVED** [review]

**Pull request merged**